app/lib/swagger_docs.rb

```diff
@@ -1180,6 +1182,93 @@ def schema
               },
             },
           },
+          GrossIncomeResult: {
```

Comment:
@starswan `employment_details` attribute is missing in the response
- assessment.gross_income.employment_details
- assessment.partner_gross_income.employment_details